apps/api/src/server/applications/application/documents/document.validators.js

```diff
@@ -31,6 +31,8 @@ const validateAllDocumentsExist = async (documentGuids) => {
 export const validateDocumentsToUploadProvided = composeMiddleware(
 	body('[]').notEmpty().withMessage('Must provide documents to upload'),
 	body('*.documentName').exists().withMessage('Must provide a document name'),
+	body('*.documentType').exists().withMessage('Must provide a document type'),
+	body('*.documentSize').exists().withMessage('Must provide a document size'),
```

Comment:
(suggestion) I think if you wanted to, you could validate that this is a number (not a string for instance)

Reply:
good call, I will try and remember for next time